raiden-dapp/tests/unit/components/channels/channel-list.spec.ts

```diff
@@ -84,20 +85,21 @@ describe('ChannelList.vue', () => {
 
   test('disable actions while "busy", show spinner on "selectedChannel"', async () => {
```

Comment:
Feels like this test is testing multiple things in one test case. It is pretty difficult to make sense of the things being tested. I would consider splitting it up in separate test cases. I think the fact that there has to be comments in the tests indicates that they perhaps should be split. ;)

Reply:
I know it was like this from before, but if you find the time it would be nice to clean it up a bit.

Reply:
@taleldayekh  I feel the things checked are part of the same transaction, the channel is first made `busy` => `true` and then made `busy` => `false` which is the same transaction as we perform it in the UI, its just that `busy` => `false` happens automatically for us without human action but on the UI it is part of the same transaction, so should remain in the same test.
What I missed is changing the name of the test which has become misleading and I am changing it.

Reply:
Ok, understood. Then go ahead and merge it. Good job!